RN36727 - Updated User-defined APIs docs with CORS update (#1650)
SkylineCommunications/dataminer-docs: user-guide/Advanced_Modules/User Defined APIs/UD_APIs_Troubleshooting.md

## Issues when triggering user-defined APIs

- **Access to XXX at 'XXX' from origin 'XXX' has been blocked by CORS policy: No 'Access-Control-Allow-Origin' header is present on the requested resource.**

  User-defined APIs don't set CORS headers for safety reasons. It is not safe to use a user-defined API from a web client. See [Triggering an API](xref:UD_APIs_Triggering_an_API#cors) for more info.

> If issues occur when you trigger a user-defined API, follow the steps below to resolve them.

Reviewer: S: new item here

Author: Added subtitle

## Issues when triggering user-defined APIs

- **Access to XXX at 'XXX' from origin 'XXX' has been blocked by CORS policy: No 'Access-Control-Allow-Origin' header is present on the requested resource.**

>   For safety reasons, user-defined APIs do not set CORS headers. It is not safe to use a user-defined API from a web client. See [Triggering an API](xref:UD_APIs_Triggering_an_API#cors).

Reviewer 2: ```suggestion
  For safety reasons, user-defined APIs does not set CORS headers. It is not safe to use a user-defined API from a web client. See [Triggering an API](xref:UD_APIs_Triggering_an_API#cors).
```

>   User-defined APIs don't set CORS headers for safety reasons. It is not safe to use a user-defined API from a web client. See [Triggering an API](xref:UD_APIs_Triggering_an_API#cors) for more info.

Reviewer 3: Avoid contractions (e.g. "don't") in the user guide. I'd propose to rephrase this as follows:

```suggestion
  For safety reasons, user-defined APIs do not set CORS headers. It is not safe to use a user-defined API from a web client. See [Triggering an API](xref:UD_APIs_Triggering_an_API#cors).
```
